### PR TITLE
Add postgres support

### PR DESCRIPTION
**Pull request by dimitrylukashov**
`ddldump` can now be run against postgres databases, in addition to mysql. Will print error message if a different dialect.

JIRA link: https://percolate.atlassian.net/browse/DTST-216

### CODE REVIEW

**Comment by lra** [comment]
do you mind adding a postgresql test on CCI where you load a sql table, ddldump it and make sure the result is the same?

If you want I can migrate the circleci config to v2 so you have access to easy to use psql containers.

**Comment by lra** [comment]
Also I guess you need to add somewhere in the doc that in the current state, you need a psql client installed.

**Comment by dimitrylukashov** [comment]
@lra wouldn't we want a test like that for mysql? Yes, please migrate to v2 CCI.

Do the docs specify that you need a mysql client as well or as a dependency? Also, its curious that the docs already mention Postgres even if it didn't work with postgres.

**Comment by lra** [comment]
Migrated to v2, you can now use any basic mysql or psql container for your tests.

**Comment by lra** [comment]
@dimitrylukashov it might help: https://github.com/lra/docker/blob/master/.circleci/config.yml#L37-L61

**Comment by dimitrylukashov** [comment]
@lra I added a a postgres and mysql test. Can you take a look again? And where do we need to add that this relies on `pg_dump`? In the readme, `setup.py`, or somewhere else?

**Comment by lra** [comment]
everywhere you can, even in the binary (e.g when you launch it with a PG db configured, it should throw an error telling you that you need to install the pg clients package)

File: ci/terraform/postgresql.tf [hunk 1]
```diff
@@ -0,0 +1,21 @@
+provider "postgresql" {
```
Comment (lra) [inline ci/terraform/postgresql.tf]: I think you can ditch those

File: ddldump/main.py [hunk 2]
```diff
@@ -79,13 +80,82 @@ def get_table_ddl(engine, table):
...
+    table_ddl = None
+
+    if engine.name == 'mysql':
+        result = engine.execute('SHOW CREATE TABLE `{}`;'.format(table))
+        row = result.first()
+        table_ddl = row[1] + ';'
+    elif engine.name == 'postgresql':
+        table_ddl = _show_create_postgresql(engine, table)
+    else:
+        print "ddldump not support the {} dialect.".format(engine.name)
```
Comment (lra) [inline ddldump/main.py]: `"ddldump not support`

**Comment by dimitrylukashov** [comment]
@lra if you try to run this script against psql without `postgresql-client` installed it already prints out the following error message:
```
Error: You must install at least one postgresql-client-<version> package.
```

**Comment by lra** [comment]
Released 0.3: https://pypi.python.org/pypi/ddldump

**Pull request merged**